pkg/core/contract_listener.go

```diff
@@ -31,12 +31,13 @@ type ContractListener struct {
 	Namespace string                   `ffstruct:"ContractListener" json:"namespace,omitempty" ffexcludeinput:"true"`
 	Name      string                   `ffstruct:"ContractListener" json:"name,omitempty"`
 	BackendID string                   `ffstruct:"ContractListener" json:"backendId,omitempty" ffexcludeinput:"true"`
-	Location  *fftypes.JSONAny         `ffstruct:"ContractListener" json:"location,omitempty"`
+	Location  *fftypes.JSONAny         `ffstruct:"ContractListener" json:"location,omitempty" ffexcludeinput:"true"`
 	Created   *fftypes.FFTime          `ffstruct:"ContractListener" json:"created,omitempty" ffexcludeinput:"true"`
-	Event     *FFISerializedEvent      `ffstruct:"ContractListener" json:"event,omitempty" ffexcludeinput:"postContractAPIListeners"`
-	Signature string                   `ffstruct:"ContractListener" json:"signature" ffexcludeinput:"true"`
+	Event     *FFISerializedEvent      `ffstruct:"ContractListener" json:"event,omitempty" ffexcludeinput:"true"`
```

Comment:
I don't understand how this is a safe change @EnriqueL8 
Surely in a point release we need to accept the old `POST` identically, and map it internally to the new way we're storing things?

Reply:
We still accept it but this just means we are excluding it from the Swagger